ktor-client/ktor-client-core/js/src/io/ktor/client/engine/js/JsClientEngine.kt

```diff
@@ -54,19 +54,25 @@ internal class JsClientEngine(override val config: HttpClientEngineConfig) : Htt
         )
     }
 
+    // Adding "_capturingHack" to reduce chances of JS IR backend to rename variable,
```

Comment:
Does the chance depend on the length or existence of a character `_` or something?

Reply:
All of it contributes to chance. It is long and contains "hack" to prevent accidental renaming it back.